specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -5592,11 +5580,7 @@
         "assetId": {
           "readOnly": true,
           "type": "string",
-          "description": "Specifies the unique of the dedicated physical machine on which the dedicated host resides."
-        },
-        "capacity": {
```

Comment:
so capacity does not exist at the begining?  Please explain this is not a breaking change.

Reply:
We never exposed these fields to the customers so it is not breaking.